tests/unit/acl.tcl

```diff
@@ -217,6 +217,25 @@ start_server {tags {"acl"}} {
         set e
     } {*NOPERM*}
 
+    test {ACLs set can include subcommands, if already full command exists} {
+        r ACL setuser bob +pfcount|hll
+        set cmdstr [dict get [r ACL getuser bob] commands]
+        assert_equal {-@all +pfcount|hll} $cmdstr
```

Comment:
I don't like the fact that this test uses `hll` as if it's a sub-command of `pfcount`
I would prefer to convert this test to check some command that really does have a sub-command.
how about `OBJECT REFCOUNT` or `MEMORY USAGE`?